packages/search-ui/src/SearchDriver.ts

```diff
@@ -136,7 +136,7 @@ class SearchDriver {
     (expansions?: Record<string, unknown>) => string
   >;
   startingState: SearchState;
-  apiConnector: APIConnector;
+  apiConnector?: APIConnector;
```

Comment:
apiConnector is required here. The tests are wrong and will need updating. LMK if you want me to step in to update

Reply:
I tried and found it requires a lot change and I'm not familiar with convention of working with non-strict TS. Personally I would expect all type defs to be strict but that may introduce breaking changes.

So it would be great if you can do it.

Reply:
done and updated tests.